nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -164,6 +161,71 @@ public ValidationResult validate(final String subject, final String input, final
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    public static final PropertyDescriptor MIN_IDLE = new PropertyDescriptor.Builder()
+            .name("Minimum Idle Connections")
```

Comment:
I saw your comment about `.name`. I know the existing properties in DBCPConnectionPool do not use `displayName`, but that is only because they are from before the change in standards.

Can you move the new `name` property values to `displayName`, and set `name` to something like `dbcp-min-idle-conns`?